ui/app/templates/components/link-status.hbs

```diff
@@ -0,0 +1,22 @@
+{{#if this.status}}
+  <div class="navbar-status {{this.bannerClass}}">
+    <Icon @name="info" />
+    <p data-test-link-status>
+      {{#if (eq this.status "connected")}}
+        This self-managed Vault is linked to the
+        <a href="https://portal.cloud.hashicorp.com/sign-in" target="_blank" rel="noopener noreferrer">
+          HashiCorp Cloud Platform
```

Comment:
Do these need a period at the end of them? From what I understand any message or text in the UI that is a sentence and is user facing needs to end in a period.

Reply:
I copied the text directly from the figma designs. It does seem strange that just this one is missing a period so possibly a mistake in the designs. I will add a period for consistency.